testapi.pm

```diff
@@ -1199,9 +1199,11 @@ Uses C<$testapi::password> if no string is given.
 
 =cut
 sub type_password {
-    my ($string) = @_;
+    my ($string, %args) = @_;
     $string //= $password;
-    type_string $string, max_interval => 100, secret => 1;
+    $args{max_interval} //= 100;
+    $args{secret} //= 1;
```

Comment:
I know. So? I am just setting sane defaults in case any args are given as argument. Would you recommend I silently overwrite $args{secret} with "1" in case the user sets something different?